src/libs/usePolyfill/index.js

```diff
@@ -0,0 +1,18 @@
+import shouldPolyfill from './shouldPolyfill';
```

Comment:
I think let's avoid using the hook naming convention for the libs. We can just keep it `polyfill`?

Also do you think we should name this intl-polyfill?

Reply:
To clarify I didn’t mean shouldUsePolyfill. I meant the directory name usePolyfill